util/args.go

```diff
@@ -57,6 +58,7 @@ fragmentation for the first data packet and the rest
 		"pattern",
 		"bypass DPI only on packets matching this regex pattern; can be given multiple times",
 	)
+	flag.BoolVar(&args.DnsIpv6Enabled, "dns-ipv6-enabled", true, "enable DNS IPv6 resolution support")
```

Comment:
Or something like  `dns-ipv4-only`

Reply:
I think `dns-ipv4-only` would be better.